float32 effort
float32 velocity
float32 velocity_variation
float32 leg_x
float32 leg_y
float32 leg_z
